docs/howtos.rst

```diff
@@ -0,0 +1,53 @@
+Flax HOWTOs
+===========
+
+Flax aims to be a thin library of composable primitives. You compose
+these primites yourself into a training loop that you write and is fully under
+your control. You have full freedom to modify the behavior of your training loop,
+and you should generally not need special library support to implement
+the modifications you want.
+
+To help you get started, we show some sample diffs, which
+we call "HOWTOs". These HOWTOs show common modifications to training loops. For instance,
+the HOWTO for ensembling learning demonstrates what changes should be made to
+the standard MNIST example in order to train an ensemble of models on
+multiple devices.
+
+Note that these HOWTOs do not require special library support, they just
+demonstate how assembling the JAX and Flax primitives in different ways
+allow you to achieve various training loop modifications.
```

Comment:
```suggestion
allows you to achieve various training loop modifications.
```

Also, "achieve various training loop modifications" sounds a bit strange. I don't think you "achieve" a modification, you "make" it.

Reply:
You're right, I fixed that.